ui/app/src/components/FormsEngine/controls/Label.tsx

```
@@ -0,0 +1,37 @@
+
+export function Label(props: ControlProps) {
+	const { field } = props;
+
+	// region properties/validations
+	const labelText = field.properties?.text?.value;
+	const renderAsHTML = field.properties?.renderAsHTML?.value === true;
+	// endregion
+
+	return renderAsHTML ? (
+		<>
+			<span dangerouslySetInnerHTML={{ __html: labelText || field.defaultValue || '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)